src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -797,7 +823,7 @@ private async Task<TextDocumentEdit[]> RemapVersionedDocumentEditsAsync(TextDocu
             foreach (var entry in documentEdits)
             {
                 var virtualDocumentUri = entry.TextDocument.Uri;
-                if (!CanRemap(virtualDocumentUri))
+                if (!_languageServerFeatureOptions.IsVirtualDocumentUri(virtualDocumentUri))
```

Comment:
Personally I liked the explicitness of `CanRemap` and the way it centralized the logic, even if it's currently a single call.

Reply:
Interesting.. to me, calling the service does a better job of "centralizing", as previously `CanRemap` simply copied logic that already existed elsewhere, but I can understand the naming isn't as nice. Will add a comment.